addons/website_sale/models/sale_order.py

```diff
@@ -302,7 +302,15 @@ def _update_address(self, partner_id, fnames=None):
             delivery_method = self._get_preferred_delivery_method(delivery_methods)
             self._set_delivery_method(delivery_method)
 
-    def _cart_add(self, product_id: int, quantity: float = 1.0, *, uom_id: int | None = None, **kwargs) -> dict:
+    def _cart_add(
+        self,
+        product_id: int,
+        quantity: float = 1.0,
+        *,
+        uom_id: int | None = None,
+        combo_item_products: list = [],
```

Comment:
It's a bad practice to [use a list as default](https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/)
```suggestion
        combo_item_products: list | None = None,
```